### ISSUE TITLE
Synchronize Directories - Asimmetric checkbox state is not saved

### ISSUE THREAD

**Issue by VictorTrifu** [doublecmd/doublecmd#1854]
### Version

Double Commander
Version: 1.1.18 gamma
Revision: 568
Commit: c45365d1b
Build date: 2024/09/21
Lazarus: 3.5.0.0
FPC: 3.2.2
Platform: x86_64-Win64-win32/win64
OS version: Windows 10 Home 22H2 x86_64

### Description

In the Synchronize Directories dialog the state of the asimmetric checkbox is not saved, so that the next time you open the dialog is checked. The other check boxes like "subdirs" "by content" "ignore" date are remembered fine.

**Comment by alexx2000** [comment]
Not a bug, [see](https://doublecmd.github.io/doc/en/configxml.html#manualed) attribute `Save` from `<SyncDirs><Asymmetric>`.

**Comment by VictorTrifu** [comment]
Thank you very much Alex for you quick response! I have already tested it and it works like a charm! Double Commander is very cool, I switched from Total Commander to it! 

I have never seen this kind of engagement and speed of response even in the paid software with paid technical support. Actually they leave the you there hanging if they don't like your question.

I'm using synchronized directories to backup my data and I still have some issues with it. So I'm going to create some more posts. Thank you for solving this problems.
